### PR TITLE
[ID] Add `Play Style/Single-tapping`

### PR DESCRIPTION
**Pull request by Fhoxydeo**

### CODE REVIEW

File: wiki/Play_style/Single-tapping/id.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  - tap
+  - single tap
+  - singletap
+  - play style
+  - gaya bermain
+---
+
+# Single-tapping
+
+**Single-tapping** adalah gaya tapping untuk [osu!](/wiki/Game_mode/osu!) yang dimana pemain menekan circle, menggeser slider, dan memutar spinner dengan hanya menggunakan satu tombol di keyboard mereka (`Z` atau `X` secara default). Gaya ini paling umum di antara pemain dari semua tingkat keahlian dan paling mudah untuk diadaptasi.
```
Comment (hikikuman) [inline wiki/Play_style/Single-tapping/id.md:14]:
```suggestion
**Single-tapping** adalah gaya tapping pada mode permainan [osu!](/wiki/Game_mode/osu!) yaitu pemain menekan circle, menggeser slider, dan memutar spinner dengan hanya menggunakan satu tombol di keyboard mereka (secara default adalah tombol `Z` atau `X`). Gaya ini merupakan gaya yang paling umum digunakan oleh para pemain dari semua tingkat kesulitan dan yang paling mudah untuk diadaptasi.
```

File: wiki/Play_style/Single-tapping/id.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+Meski namanya single-tapping, sebagian besar pemain single-tap tidak selalu menekan dengan satu tombol.Sebagian besar lebih suka menekan semua secara singletap kecuali [*stream*](/wiki/Beatmap/Pattern/Stream), [*stack*](/wiki/Mapping_Techniques/Stack), dan *burst* yang cepat. Dalam hal ini mereka akan sering memilih untuk [*alternate*](/wiki/Play_style/Alternating) dan/atau [*double tap*](/wiki/Play_style/Double_tapping).
```
Comment (hikikuman) [inline wiki/Play_style/Single-tapping/id.md:16]:
```suggestion
Meski namanya single-tapping, sebagian besar pemain single-tap tidak selalu menekan hanya dengan satu tombol saja. Sebagian besar pemain lebih suka menekan semua secara single-tap kecuali [stream](/wiki/Beatmap/Pattern/Stream), [stack](/wiki/Mapping_Techniques/Stack), dan burst stream yang cepat. Dalam hal ini, para pemain akan lebih memilih untuk [alternate](/wiki/Play_style/Alternating) dan/atau [double tap](/wiki/Play_style/Double_tapping).
```
gaperlu dibuat garis miring, cmiiw.

File: wiki/Play_style/Single-tapping/id.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+## Argumen
+
+Argumen yang mendukung single-tapping biasanya berpendapat bahwa lebih mudah untuk konsisten karena -untuk semua tujuan dan maksud- hanya satu jari saja yang bermain. Dan biasanya inilah alasannya(sadar atau tidak) beberapa pemain baru menggunakan metode ini.
```
Comment (hikikuman) [inline wiki/Play_style/Single-tapping/id.md:20]:
```suggestion
Argumen yang mendukung single-tapping biasanya berpendapat bahwa lebih mudah untuk konsisten karena -untuk semua tujuan dan maksud- hanya satu jari saja yang bermain. Dan biasanya inilah alasannya (sadar atau tidak) beberapa pemain baru menggunakan metode ini.
```

File: wiki/Play_style/Single-tapping/id.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+Dengan cara yang sama, banyak pemain beranggapan bahwa single-tapping lebih lambat karena ada penundaan yang diperlukan sebelum Anda dapat menekan jari yang sama , sementara yang lain beranggapan dengan hanya menggunakan satu jari dapat menyebabkan pengguna kelelahan dalam waktu yang cukup cepat.
```
Comment (hikikuman) [inline wiki/Play_style/Single-tapping/id.md:22]:
```suggestion
Dengan cara yang sama, banyak pemain beranggapan bahwa single-tapping lebih lambat karena ada penundaan yang diperlukan sebelum kamu dapat menekan jari yang sama, sementara yang lain beranggapan dengan hanya menggunakan satu jari dapat menyebabkan pengguna kelelahan dalam waktu yang cukup cepat.
```

**Review by hikikuman — CHANGES_REQUESTED** [review]

File: wiki/Play_style/Single-tapping/id.md (hunk 2 above)
Comment (Fhoxydeo) [reply wiki/Play_style/Single-tapping/id.md:16]: kalo ama niva kemarin dikasih garis miring

File: wiki/Play_style/Single-tapping/id.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+  - tap
+  - single tap
+  - singletap
+  - play style
+  - gaya bermain
+---
+
+# Single-tapping
+
+**Single-tapping** adalah gaya tapping pada mode permainan [osu!](/wiki/Game_mode/osu!) yaitu pemain menekan circle, menggeser slider, dan memutar spinner dengan hanya menggunakan satu tombol di keyboard mereka (secara default adalah tombol `Z` atau `X`). Gaya ini merupakan gaya yang paling umum digunakan oleh para pemain dari semua tingkat kesulitan dan yang paling mudah untuk diadaptasi.
```
Comment (mousesalt) [inline wiki/Play_style/Single-tapping/id.md:14]:
```suggestion
**Single-tapping** adalah gaya tapping pada mode permainan [osu!](/wiki/Game_mode/osu!) yaitu pemain menekan circle, menggeser slider, dan memutar spinner dengan hanya menggunakan satu tombol di keyboard mereka (secara default adalah tombol `Z` atau `X`). Gaya ini paling umum digunakan oleh para pemain dari semua tingkat kesulitan dan yang paling mudah untuk diadaptasi.
```

Comment (mousesalt) [reply wiki/Play_style/Single-tapping/id.md:14]: unused iteration

File: wiki/Play_style/Single-tapping/id.md [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+
+Meski namanya single-tapping, sebagian besar pemain single-tap tidak selalu menekan dengan satu tombol.Sebagian besar lebih suka menekan semua secara singletap kecuali [*stream*](/wiki/Beatmap/Pattern/Stream), [*stack*](/wiki/Mapping_Techniques/Stack), dan *burst* yang cepat. Dalam hal ini mereka akan sering memilih untuk [*alternate*](/wiki/Play_style/Alternating) dan/atau [*double tap*](/wiki/Play_style/Double_tapping).
```
Comment (mousesalt) [inline wiki/Play_style/Single-tapping/id.md:16]:
```suggestion
Meski namanya single-tapping, sebagian besar pemain single-tap tidak selalu menekan dengan satu tombol. Sebagian besar pemain lebih suka menekan semua secara single-tap kecuali [*stream*](/wiki/Beatmap/Pattern/Stream), [*stack*](/wiki/Mapping_Techniques/Stack), dan *burst* yang cepat. Dalam hal ini, pemain akan sering memilih untuk melakukan [*alternate*](/wiki/Play_style/Alternating) dan/atau [*double tap*](/wiki/Play_style/Double_tapping).
```

File: wiki/Play_style/Single-tapping/id.md [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+
+## Argumen
+
+Argumen yang mendukung single-tapping biasanya berpendapat bahwa lebih mudah untuk konsisten karena -untuk semua tujuan dan maksud- hanya satu jari saja yang bermain. Dan biasanya inilah alasannya (sadar atau tidak) beberapa pemain baru menggunakan metode ini.
```
Comment (mousesalt) [inline wiki/Play_style/Single-tapping/id.md:20]:
```suggestion
Argumen yang mendukung single-tapping biasanya berpendapat bahwa lebih mudah konsisten karena—untuk semua tujuan dan maksud—hanya satu jari saja yang bermain. Dan biasanya inilah alasannya (sadar atau tidak) banyak pemain baru menggunakan metode ini.
```

Comment (mousesalt) [reply wiki/Play_style/Single-tapping/id.md:20]: reviewed, does think it a bit confused

**Review by hikikuman — APPROVED** [review]
👍

**Review by mousesalt — APPROVED** [review]

**Review by wowcake — APPROVED** [review]

**Review by TPGPL — APPROVED** [review]

**Pull request merged**